lib/salestation/rspec/glia_input_validation_error_matcher.rb

```diff
@@ -24,10 +25,16 @@ def with_type(*types)
         self
       end
 
+      def with_message(*messages)
```

Comment:
We actually already have a way to check messages. For example https://github.com/salemove/phone-number-manager/blame/master/spec/phone_number_manager/delete_voice_mailbox_spec.rb#L66

This also lets us assert the error details like here for example: https://github.com/salemove/phone-number-manager/blame/master/spec/phone_number_manager/update_incoming_phone_number_spec.rb#L259

Reply:
I see. I think this change is still valuable as it adds the functionality to the `glia_input_validation_error` matcher. The examples you shown, doesn't use the matcher.